app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,20 @@ def show
     id = params[:id]
     @post = Post.find(id)
   end
+
+  def new; end
+
+  def create
+    user = User.find(params[:user_id])
+    text = params[:text]
+    title = params[:title]
+
+    new_post = Post.new(title:, text:, author: user)
+
+    if new_post.save
+      redirect_to user_post_url(user, new_post)
+    else
+      render :new
+    end
+  end
```

Comment:
- [OPTIONAL] It will be nice to add strong parameters to your controllers as this will make your app more secure. Something like so;
```
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```